elfloader-tool/src/arch-riscv/crt0.S

```diff
@@ -100,16 +106,17 @@ hsm_switch_hart:
 
 
 _start1: /* a0 must hold current hard ID passed by bootloader */
-
+         /* a1 must hold dtb address passed by bootloader */
```

Comment:
`a1` is overwritten with the value from `s2` line 125, this must be removed.

Reply:
Removed